### PR TITLE
CSS

### PR DESCRIPTION
**Pull request by marioylink11**

### CODE REVIEW

File: CSS/2.1/index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+</head>
+
+<body>
+	<!--[if lte IE 9]>
+	<p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience and security.</p>
+	<![endif]-->
+
+	<!-- Add your site or application content here -->
+	<header>
+		<p>Hello world! This is HTML5 Boilerplate.</p>
```
Comment (troianoandres) [inline CSS/2.1/index.html]: Should this be a `p` tag? What might be more descriptive for the page?

File: CSS/2.1/index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	</header>
+	<!--
+		<script src="js/vendor/modernizr-{{MODERNIZR_VERSION}}.min.js"></script>
+		<script src="https://code.jquery.com/jquery-{{JQUERY_VERSION}}.min.js" integrity="{{JQUERY_SRI_HASH}}" crossorigin="anonymous"></script>
```
Comment (troianoandres) [inline CSS/2.1/index.html]: Please remove comments if not needed

File: CSS/2.1/index.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+	<nav>
+		<ul>
+			<li>bla</li>
+			<li>bla</li>
+			<li>bla</li>			
+		</ul>
+	</nav>
+	
+	<section>
+		<h1>Section</h1>
```
Comment (troianoandres) [inline CSS/2.1/index.html]: Which is the meaning of `h1`. Should it be used here? Is the right heading level?

File: CSS/2.2/style.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
+body{
+    width:100%;
+	height: 100%;
```
Comment (troianoandres) [inline CSS/2.2/style.css]: Try to avoid different tabbings, keep consistent across all lines please :)

File: CSS/2.2/style.css [hunk 5]
```diff
@@ -0,0 +1,57 @@
+body{
+    width:100%;
+	height: 100%;    
+    height:auto; 
+}
+
+header#MainHeader{
```
Comment (troianoandres) [inline CSS/2.2/style.css]: CSS selectors must be written in snake case. This specific selector should be `main-header`

File: CSS/2.2/style.css [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+ } 
+
+ /*clear: none|left|right|both|initial|inherit;*/
+nav {   
+  float: left;
+  height: 100%;
+min-height: 75vh;  
+  width:15%;
+  margin: 0 20px;
+  top: 0px;
```
Comment (troianoandres) [inline CSS/2.2/style.css]: When using `0px` you should refer to it as `0` instead

**Comment by troianoandres** [comment]
Seems that this PR has also the changes for the other branch. When you open a new branch please go back to master before opening the new branch

File: CSS/2.3/style.css [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+nav {   
+	float: left;
+	height: 100%;
+	min-height: 75vh;  
+	width:15%;
+	margin: 0 20px;
+	top: 0;
+	background-color: lightblue;
+	bottom: 0;
+	clear;
```
Comment (troianoandres) [inline CSS/2.3/style.css]: Clear?

File: CSS/2.5/2.5.2/style.css [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+.block{
+	float:left;
+	height:40%;
+	width:30%;	
+	display:block;	
+}
+
+
+.topRow{
```
Comment (troianoandres) [inline CSS/2.5/2.5.2/style.css]: Classes are not expected to be written like this

File: CSS/2.5/2.5.2/style.css [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+	margin:20px 20px;
```
Comment (troianoandres) [inline CSS/2.5/2.5.2/style.css]: This can be shortened

File: CSS/2.5/2.5.2/style.css [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+	
+}
+
+.bottomRow{
+	margin:20px 20px;
```
Comment (troianoandres) [inline CSS/2.5/2.5.2/style.css]: Same here

File: CSS/2.3/style.css [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+    height: auto;
+	text-align: center;
+    background-color: orange;
+    margin: 20px;        
+ } 
+
+ /*clear: none|left|right|both|initial|inherit;*/
+nav {   
+	float: left;
+	height: 100%;
```
[the remark this answers is not in the document]
Comment (marioylink11) [reply CSS/2.3/style.css]: if I don't set `width` and `height` in `nav`, `section` and `aside`, the respectives height would be their content's height

File: CSS/2.3/index2.3.html [hunk 12]
```diff
@@ -27,6 +27,13 @@
 		<h1>Main Header h1</h1>
 		<p>Hello world! This is HTML5 Boilerplate.</p>
 	</header>
+	<!--
+		<script src="js/vendor/modernizr-{{MODERNIZR_VERSION}}.min.js"></script>
+		<script src="https://code.jquery.com/jquery-{{JQUERY_VERSION}}.min.js" integrity="{{JQUERY_SRI_HASH}}" crossorigin="anonymous"></script>
+		<script>window.jQuery || document.write('<script src="js/vendor/jquery-{{JQUERY_VERSION}}.min.js"><\/script>')</script>
```
Comment (troianoandres) [inline CSS/2.3/index2.3.html]: Remove this, is not good practice to keep unneeded comments

File: CSS/2.5/2.5.2/style.css (hunk 8 above)
Comment (troianoandres) [reply CSS/2.5/2.5.2/style.css]: This should be `.top-row`. This is the standar for all css selectors.

**Pull request merged**